### DISCUSSION TITLE
Automate object emoji

### DISCUSSION THREAD

**Discussion by Madis0** [C1710/blobmoji#75] [Ideas]
Would it be feasible to take a set of character images, apply a script filter (e.g. remove gradients and borders) and get a decent image result? 

This could work with objects, body parts and symbols, but not people or emotions.

Pros:
- Overall styles can be updated by Google 
- Less original work to be done (more focus on _blobs_ )

Cons: 
- have to wait for a beta/final release to get the images 
- one filter might not fit all images

**Comment by C1710** [comment]
Yeah, I already made some script that did this thing. It was just not good.  
Anyhow, it might be something to look into, once I have the new rendering system complete, because for rendering the emojis, they already get converted into a SVG tree, so it should be relatively easy to do these transformations.  
However the main issue is that the style often doesn't fit on a level that's not auto-fixable.

**Comment by C1710** [comment]
In combination with https://github.com/C1710/blobmoji/issues/58#issuecomment-650631867 this could be actually doable now.  
I already have a system that aligns the colors to a predefined palette (although it doesn't work very well at the moment), so it might be possible to do something like this in a very easy way.

**Comment by Secret-chest** [comment]
With Android 12, there are no more gradients or borders.
